response_operations_ui/templates/conversation-view.html

```diff
@@ -0,0 +1,52 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+
+{% block main %}
+
+
+
+<h1 class="saturn" name="page-messages-title">{{ messages[0].get('subject') }}</h1>
+
+<div class="secure-message-conversation block">
+    <div class="secure-message-sent-message-meta">
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Survey: </span><span id="survey">{{ messages[0].get('survey') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">RU Ref: </span><span id="ruref">    {{ messages[0].get('ruref') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Business: </span><span id="ru">{{ messages[0].get('business') }}</span></div>
+         <div class="secure-message--recipient venus"><span class="sm-form-recipient-label">To: </span><span id="recipient">    {{ messages[0].get('to') }}  </span></div>
+
+    </div>
+    {% for msg in messages %}
+    {% if msg.get('internal') == True %}
+    <div id="smC1M1" class="secure-message-sent-message">
+        <div class="secure-message-sent-message-meta sm-from-ons">
+    {% else %}
+    <div id="smC1M2" class="secure-message-sent-message">
```

Comment:
- What do these id's mean on the divs?
- Why do the divs have different ids? Can they be the same and just vary the class on the inner div?

Reply:
Divs need a different ID because they represent differt kind of messages. One if for internal another one for external. It s justa convention for allow the Slenium test to diferenciate between them.

Reply:
Thanks @atrozzo 
@ryangrundy7 can we use a common outer div and just have a specific id on the inner div?